spec/lib/api/2.0/skus-spec.js

```diff
@@ -77,11 +77,13 @@ describe('Http.Api.Skus.2.0', function() {
                 }
             }
         ],
+        ibms: [],
         autoDiscover: "false",
         identifiers: [],
         createdAt: '010101',
         updatedAt: '010101',
         tags: [],
```

Comment:
@jasl8r This object is a mock representing the actual node document stored in Mongo.  This data is passed to this template (which we refer to as a view) as render context : https://github.com/RackHD/on-http/blob/master/data/views/node.2.0.json.  Since we store an array of tags in the node document, the mock node object includes tags as an array.  Looking at this just now, I noticed the view doesn't actually use the tags array at present.  It just presents a link to the the route that returns an array of tags for a given node so ultimately tags don't affect the behavior of this test.